_posts/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins.md

```diff
@@ -0,0 +1,336 @@
+To get indexes from a remote cluster, call `datasource1`, as shown in the following image:
+<img src="/assets/media/blog-images/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins/blog_example_remote_cluster.png">
+
+
+Refer to the [blog_example_plugin](https://github.com/zhyuanqi/OpenSearch-Dashboards/tree/blog_example/examples/blog_example_plugin) repository for additional details about the example plugin code. 
+
+## Developing, testing, and debugging
+
+Now that you've learned how to integrate multiple data sources with OpenSearch Dashboards plugins, let's explore some advanced concepts and techniques.
+
+### Example plugins for multiple data sources
+
+To provide a more intuitive and direct experience, we've included an example plugin for developers to get started quickly with the components. You can enable the example plugins by running OpenSearch Dashboards with the following CLI command:
+```
+yarn start --run-examples
+```
+
+With OpenSearch Dashboards running, navigate to the page and select the left menu. Under OpenSearch Dashboards, open **Developer Examples**, then select **Multiple Data Sources Integration**, as shown in the following image. On the right side, you'll find a list of pickers, each with a brief introduction and properties listed below. Feel free to experiment with the example plugin.
+
+<img src="/assets/media/blog-images/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins/multi_selectable.png">
+
+### Data sources as saved objects
+
+The data source is persisted as a saved object in the OpenSearch Dashboards system index. Upon creation, it is assigned a UUID as a unique identifier, and metadata such as the remote cluster version and associated plugins is gathered. This automatically generated data, along with user-provided information like the title, endpoint, and authentication methods, is stored as attributes of the data source object. Refer to the [related code](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source/common/data_sources/types.ts#L8) for the current object schema. 
+
+
+## List of supported plugins/features
+
+Starting from version 2.14, the following external OpenSearch Dashboards plugins support multiple data sources: 
+
+* Index Management
+* Machine Learning
+* Search Relevance
+* Anomaly Detection
+* Maps
+* Security
+* Notification
+* Query Workbench
+* Trace Analytics 
+
+Multiple data sources functionality is also now available with Time Series Visual Builder (TSVB), Vega and Region Map visualization types.
+
+## Frequently asked questions 
+
+Q: Is any UI component provided by `dataSourceManagemet` used in core OpenSearch?
+A: A data source selector is now used in Dev Tools, the tutorial, and on the saved object management page. You can see the component after setting `data_source.enabled` to `true` in your `opensearch_dashbards.yml` file.
```

Comment:
Should "saved object management" be formatted as a UI element (the name of the page)?

Reply:
got it